src/platform/code-editor/code-editor.component.ts

```diff
@@ -591,6 +591,58 @@ export class TdCodeEditorComponent implements OnInit, AfterViewInit, ControlValu
     return loadPromise;
   }
 
+  showFullScreenEditor(): void {
+    const codeEditorElement: HTMLDivElement = this._editorContainer.nativeElement as HTMLDivElement;
+    const fullScreenMap: Object = {
+      'requestFullscreen': () => codeEditorElement.requestFullscreen(),
+      'webkitRequestFullscreen': () => codeEditorElement.webkitRequestFullscreen(),
+      'msRequestFullscreen': () => (<any>codeEditorElement).msRequestFullscreen(),
+      'mozRequestFullScreen': () => (<any>codeEditorElement).mozRequestFullScreen(),
+    };
+
+    for (const handler of Object.keys(fullScreenMap)) {
+      if (codeEditorElement[handler]) {
+        fullScreenMap[handler]();
+      }
+    }
+  }
+
+  exitFullScreenEditor(): void {
```

Comment:
likewise being more explicit here, like:
```
exitFullScreenEditor(): void {
    if (document.exitFullscreen) {
      document.exitFullscreen();
      // Chrome / Edge / Safari
    } else if (document.webkitExitFullscreen) {
      document.webkitExitFullscreen();
      // IE
    } else if ((<any>document).mozCancelFullScreen) {
      (<any>document).mozCancelFullScreen();
      // Firefox
    } else if ((<any>document).msExitFullscreen) {
      (<any>document).msExitFullscreen();
    }
  }
```

Reply:
Similar to previous one, i was avoiding multiple if-else conditions.
I will add proper comments here as well.